README.md

```diff
@@ -91,15 +91,18 @@ To use a release downstream, update your app's `package.json` as follows:
 
 ## Releases
 
-> TLDR: Run `yarn version` and `git push origin tagname` when your PR is ready.
+> TLDR: Run `yarn version` and `git push origin tagname` when you need to create a new release.
 
-Each PR that merges to `master` should update the `package.json` version one way or another, and ideally create a corresponding git tag. One way to do this is to run `yarn version` in your branch and push. This will:
+Each time you need to consume recent EDS changes elsewhere, you need to bump the version and create a new git release:
 
-- Prompt you for the new version number
-- Update `package.json` with the version number
-- Commit the change in a new commit
-
-To create and push a git tag, run `git tag v1.2.3 && git push origin v1.2.3`. You can skip this if the release isn't specifically needed downstream yet.
+1. To bump the package.json version, run `yarn version` in your branch, This will:
+  - Prompt you for the new version number
+  - Update `package.json` with the version number
+  - Commit the change in a new commit
+2. To create and push a git tag, do one of these:
+  - Run `git tag v1.2.3 && git push origin v1.2.3` on your branch.
```

Comment:
Running `yarn version` and responding to the prompt with the bumped version will already create the tag. So you should only have the `git push` step here. cc / @zach-ethos